.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-docker:
+    name: Release Docker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
```

Comment:
not sure about this one :thinking: probably we don't need it this way and just go with the `tag` of the commit :thinking:

Reply:
As i said, i prefer versions as it allows me to pull the image with mayor.minor tag so i can just easily pull new versions.
But as the toolkit is probably a project that doesn't need updates too often im fine with using the commit too. 
Whatever the project owner prefer imo

Reply:
It works. Stole it from Hornet and tested it here: https://hub.docker.com/r/drelectron/evm-toolkit😆